lib/msal-core/src/error/ClientConfigurationError.ts

```diff
@@ -70,6 +70,10 @@ export const ClientConfigurationErrorMessage = {
     claimsRequestParsingError: {
         code: "claims_request_parsing_error",
         desc: "Could not parse the given claims request object."
+    },
+    emptyRequestError: {
+        code: "empty_request_error",
+        desc: "Request object is required"
```

Comment:
Nit pick: Period at the end (for consistency).

Reply:
updating